client/src/components/common/CustomModal.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {
+  Modal, ModalBody, ModalFooter, ModalHeader, Button,
+} from 'reactstrap';
+import PropTypes from 'prop-types';
+
+const CustomModal = ({
+  _modal, _toggle, _button1text, _button2text, _modalTitle, _modalBody, _button2color,
```

Comment:
instead of `_button1text, _button2text, _button2color`, please switch to their names according to their functionality, like `success, error` (in this case)

Reply:
Sure. Will make the changes.